gap/Morphisms/morphisms.gi

```diff
@@ -478,8 +478,523 @@ InstallMethod( InverseGeneralMapping, "for a polygonal morphism",
 
         return InversePolygonalMorphism(polMor);
     end
+);  
+
+InstallMethod(ButterflyFaithfulMonomorphismIntoSimplicialSurface, "for two simplicial surfaces",
+    [IsSimplicialSurface, IsSimplicialSurface],
+    function(surface1, surface2)
+        local PermuteList, AllBijections, AllOneFaceIsomorphisms, MyImage, MyPreImage, MyImageCat,
+        MyPreImages, MyPreImagesCat, MyUpdateMap, SurfaceIdentityMap, RemoveElement, MyLess, 
+        VertexDegreesOfFace, MyIsInjective, MonomorphismIntoSimplicialSurface;
+        PermuteList := function(list, perm)
+            return List(list, x -> x^perm);
+        end;
+
+        AllBijections := function(list1, list2)
+            local res, AllPermutations, i, bijection, g, tmp;
+            res := [];
+            if not (Length(list1) = Length(list2)) then
+                return [];
+            else
+                AllPermutations := [];
+                for g in SymmetricGroup(list2) do
+                    Add(AllPermutations, PermuteList(list2, g));
+                od;
+                for bijection in AllPermutations do
+                    tmp := [];
+                    for i in [1..Length(list1)] do
+                        tmp[list1[i]] := bijection[i];
+                    od;
+                    Add(res, tmp);
+                od;
+                # res := List(AllPermutations, x -> IndexInducedMap(list1, x));
+                return res;
+            fi;
+        end;
+
+        MyImage := function(map, x)
```

Comment:
Do you use this method somewhere?

Reply:
Oh, yeah I will delete the unnecessary comments. Also, no I do not use the function anywhere. I will remove it.

Reply:
Actually, there are a few more functions I do not use. I will remove them as well

Reply:
I have removed everything, compiled the manual and tested everything. Locally, everything worked fine. But the tests here are failing. The errors also seem to be weird. Is there a way to restart the tests here without a "no change"-push?

Reply:
Yes, I restarted the test but nothing has changed. The error is really weird and also occurs at PR218 since yesterday.

Reply:
It's a problem from GAP, it will be resolved soon and then we can restart the checks